.sync/Files.yml

```diff
@@ -451,6 +451,28 @@ group:
     repos: |
       microsoft/mu_devops
 
+# Leaf Workflow - Pull Request Validator
+  - files:
+    - source: .sync/workflows/leaf/pull-request-formatting-validator.yml
+      dest: .github/workflows/pull-request-formatting-validator.yml
+    repos: |
+      microsoft/mu

```

Comment:
Is there a way to make the repo list a variable, so that we can reuse the list?

Reply:
Yes. I think that'd be a nice refactor. I filed a feature request here to track it, so it doesn't get lost.

https://github.com/microsoft/mu_devops/issues/121